packages/vscode-extension/CHANGELOG.md

```diff
@@ -3,14 +3,16 @@
 ## 2.6.0 - Sep 06, 2021
 
 Incremental version for Teams Toolkit with multiple bugs fixed and the following updates：
- 
+
```

Comment:
why we have change in this file?

Reply:
These are lint changes, after we merge some conflict, there are so may lint changes in out PR, wenyu will help to solve these lint changes issue in the future